libcore/src/IO/Trajectories.h

```diff
@@ -47,7 +47,7 @@ class Trajectories
     virtual ~Trajectories() = default;
     virtual void WriteHeader(long nPeds, double fps, Building * building, int seed, int count) = 0;
     virtual void WriteGeometry(Building * building)                                            = 0;
-    virtual void WriteFrame(int frameNr, Building * building)                                  = 0;
+    virtual void WriteFrame(int frameNr, int precision, Building * building)                   = 0;
```

Comment:
Might be better to have a `precision` attribute in `Trajectories` since it is the same for one data set/trajectory.

Reply:
See
77179ff9acf952fb8b13c69bb614a46cbfb18abd